frontend/public/locales/en/translation.json

```diff
@@ -975,6 +976,8 @@
   "creation.valid.numeric": "Value must be integer.",
   "Credential name": "Credential name",
   "Credential name and namespace are predefined as below for HyperShift add-on": "Credential name and namespace are predefined as below for HyperShift add-on",
+  "Credential type": "Credential type",
+  "Credential Type": "Credential Type",
```

Comment:
Can you please use only "Credential type"? Otherwise looks good.

Reply:
Thanks Kevin! Will make the change

Reply:
Updated :)